apps/cli/src/commands/send.ts

```diff
@@ -81,7 +81,6 @@ export default {
                           deviceId: opts.device || "",
                         })
                         .pipe(
-                          map(toSignOperationEventRaw),
```

Comment:
not sure why this was removed but i think it's intended that we display to the console "row" version of these